FinalEngine.Editor.ViewModels/Inspectors/EntityInspectorViewModel.cs

```diff
@@ -94,4 +113,28 @@ private void InitializeEntityComponents()
             this.componentViewModels.Add(new EntityComponentViewModel(this.messenger, this.entity, component));
         }
     }
+
+    private void IntializeComponentTypes()
+    {
+        this.categorizedComponentTypes.Clear();
+
+        var assembly = Assembly.GetAssembly(typeof(TagComponent));
+
+        if (assembly is null)
+        {
+            return;
```

Comment:
This should never be null, but if it is just throw an InvalidOperationException - I think?